### DISCUSSION TITLE
Target Fan Curve

### DISCUSSION THREAD

**Discussion by jeremystevens515** [Rem0o/FanControl.Releases#1900] [Ideas]
Set a target temperature with a +-range and the fans will automatically adjust to maintain that temperature. The real feature is obviously the algorithm used to accomplish this, as this could be very annoying if not done properly.

**Comment by Rem0o** [comment]
The "Auto" fan curve sort of does this.  It will try and hold the load temperature +- the deadband with the minimum fan speed required.

**Reply by jeremystevens515** [reply]
I'll have to try it out!

**Reply by Rem0o** [reply]
It's best suited for constant load scenarios. Less so for daily with always changing load.

**Reply by jeremystevens515** [reply]
Yeah I like the trigger curves because they stay constant up to a point, but if your temp spikes you get blasted until it reaches idle again, which is only problematic if the idle temp is set super low. I've experimented with idle being like 60 and load being 65-70. There's just the variation in the upper range missing.
